plugins/mds.py

```diff
@@ -360,9 +360,10 @@ def calculation_task(self, db_id: int) -> str:
     entity_points = []
 
     for ent_id, idx in id_to_idx.items():
-        entity_points.append(
-            {"ID": ent_id, "href": "", "point": [x for x in transformed[idx]]}
-        )
+        new_entity_point = {"ID": ent_id, "href": ""}
+        new_entity_point.update({str(i): x for i, x in enumerate(transformed[idx])})
```

Comment:
The entity attributes are sorted lexicographically. This means that dimensions will sort like this `1,11,12,2,3,4`. Number strings have to be padded with 0 to sort correctly. Other plugins outputting `entity/vector` data already do this.